### ISSUE TITLE
ScrollContainer.Refresh() doesn't seem to work properly

### ISSUE THREAD

**Issue by wenhsinhsieh** [fyne-io/fyne#1341]
### Describe the bug:
ScrollContainer.Refresh() doesn't seem to work properly

### Example code:
Changing the content of a ScrollContainer and refresh doesn't really update the screen, for example:
```go
something := drawSomething()
container := widget.NewScrollContainer(something)

updateButton := widget.NewButton("Update", func() {
	container.Content = drawSomethingElse()
	container.Refresh()
})
```
However, if I wrap the content in another container, it works as expected, for example:
```go
something := drawSomething()
innerContainer := fyne.NewContainerWithLayout(layout.NewVBoxLayout(), something)
container := widget.NewScrollContainer(innerContainer)

updateButton := widget.NewButton("Update", func() {
	innerContainer.Objects = []fyne.CanvasObject{drawSomethingElse()}
	innerContainer.Refresh()
})
```
So I doubt that something wrong with the ScrollContainer.Refresh()

### Device (please complete the following information):
 - OS: Windos
 - Version: 10
 - Go version: 1.14.6
 - Fyne version: 1.3.3

**Comment by andydotxyz** [comment]
This is fixed on develop for testing, will be in 1.4 release

**Comment by wenhsinhsieh** [comment]
Wow you fix it so fast! Thanks you very much!
